samples/ai-image-tagging/src/lib/tags.js

```diff
@@ -0,0 +1,25 @@
+import { uniq } from 'lodash';
+
+const BACKEND_URL = 'https://staging.ctffns.net/image-tagging';
```

Comment:
It will be updated to a microbackend url, right?

Reply:
this points to a micro backend in staging atm